packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -109,13 +109,15 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
     field: string,
     children: VNode[]
   ) {
+    const label = this.bindings.store.state.facets[facetId].label || field;
     return (
       <div class="flex text-sm" part="breadcrumb-wrapper">
         <span
-          class="text-on-background mr-2 flex-shrink-0"
+          title={label}
+          class="breadcrumb-label text-on-background mr-2 flex-shrink-0 overflow-ellipsis overflow-hidden"
```

Comment:
In the tailwind doc, I see that `truncate` applies `overflow-ellipsis` and `overflow-hidden` as well as `overflow-clip`, could `truncate` be used instead of `"overflow-ellipsis overflow-hidden"` here and below?